howfairis/mixins/license_mixin.py

```diff
@@ -20,6 +20,10 @@ def has_license(self):
 
         r = False
 
+        if self.repo.platform == Platform.BITBUCKET:
+            # I haven't found a good way to determine the license on BitBucket yet
+            r = False
```

Comment:
Spinned of https://github.com/fair-software/howfairis/issues/183 to see if LICENSE file is present